Template10 (Library)/Controls/HamburgerButtonInfo.cs

```diff
@@ -83,6 +79,14 @@ public bool IsEnabled
             DependencyProperty.Register(nameof(IsEnabled), typeof(bool),
                 typeof(HamburgerButtonInfo), new PropertyMetadata(true));
 
+        public string ToolTipText
+        {
+            get { return (string)GetValue(ToolTipTextProperty); }
+            set { SetValue(ToolTipTextProperty, value); }
+        }
+        public static readonly DependencyProperty ToolTipTextProperty =
+            DependencyProperty.Register("ToolTipText", typeof(string), typeof(HamburgerButtonInfo), new PropertyMetadata(null));
```

Comment:
Yes, I always try and use C#6 wherever possible, this one slipped through the net with the snippet as you said